WalletWasabi/Wallets/P2pBlockProvider.cs

```diff
@@ -316,9 +321,11 @@ private async Task NodeTimeoutsAsync(bool increaseDecrease)
 		}
 
 		// Sanity check
-		if (timeout < 32)
+		var minTimeout = Network == Network.Main ? 3 : 2;
+		minTimeout = HttpClientFactory.IsTorEnabled ? (int)Math.Round(minTimeout*1.5) : minTimeout;
```

Comment:
What is happening here? Can you elaborate on this? 

It seems like a subtle tweak.

Reply:
The minimum timeout is a double edged sword:
It's the most significant improvement on wallet load time that we will be able to make.
Setting too high and we would have to wait flat seconds sync for nothing
Setting too low and we will end up downloading all blocks from same fast nodes, against privacy.
Blocks in mainet are bigger that in testnet, so it makes sense to have a bigger minimum timeout
Because even setting a 1s min timeout is perfectly fine and blazing fast, I'm testing my other features with that.
But I download all blocks from same nodes.

I will send some data today about dl time and privacy with different timeout settings